src/components/Comments.jsx

```diff
@@ -0,0 +1,37 @@
+import { fetchCommentsByArticleID } from "../api";
+import { useEffect } from "react";
+
+const Comments = ({ article_id, comments, setComments }) => {
+  useEffect(() => {
+    if (typeof article_id !== "undefined" && article_id !== "") {
+      fetchCommentsByArticleID(article_id)
+        .then((comments) => {
+          setComments(comments);
+        })
+        .catch((err) => {
+          console.log(err);
+        });
+    }
+  }, [article_id, setComments]);
+
+  return (
+    <section className="Comments">
+      <h3>Comments</h3>
+      <ul className="comments-flex-container">
+        {comments.map((comment) => {
```

Comment:
we should also have some sort of indication if there are no comments, it would be better for the user experience if they had some visual cue that there was no comments left on this.